src/main/java/com/kodilla/ecommercee/entity/UserOrder.java

```diff
@@ -19,4 +19,17 @@
     @Id
     @GeneratedValue
     private Long Id;
+
+    private LocalDate orderMade;
+
+    private boolean resolved;
+
+    @OneToOne(cascade = CascadeType.ALL,
+            fetch = FetchType.EAGER)
+    @JoinColumn
+    private User userId;
```

Comment:
O ile w DTO operujemy na samym ID, to tutaj masz pełną relacje do Usera. WIęc można nazwać to po prostu User.
Zwróć uwagę, że DTO i baza mogą mieć nieco inny model. Dto zazwyczaj mają tylko relacje przy użyciu Id, a baza pełną